### DISCUSSION TITLE
2D meshing workflow

### DISCUSSION THREAD

**Discussion by hobum-92** [ansys/pyfluent#3064] [Q&A]
I ran the following example of '2D meshing workflow' on the Pyansys homepage.
 
import ansys.fluent.core as pyfluent
from ansys.fluent.core import examples

import_file_name = examples.download_file('NACA0012.fmd', 'pyfluent/airfoils')
meshing = pyfluent.launch_fluent(
    mode="meshing", precision='double', processor_count=2
)
two_dim_mesh = meshing.two_dimensional_meshing()

two_dim_mesh.load_cad_geometry_2d.file_name = import_file_name
two_dim_mesh.load_cad_geometry_2d.length_unit = "mm"
two_dim_mesh.load_cad_geometry_2d.refaceting.refacet = False
two_dim_mesh.load_cad_geometry_2d()

The result was the following error message:
RuntimeError: '2D Meshing' objects are inaccessible from other workflows.

I ran another example.

import ansys.fluent.core as pyfluent
from ansys.fluent.core import examples

import_file_name = examples.download_file('NACA0012.fmd', 'pyfluent/airfoils')
meshing = pyfluent.launch_fluent(
    mode="meshing", precision='double', processor_count=2
)
meshing.workflow.InitializeWorkflow(WorkflowType="2D Meshing")
meshing.workflow.TaskObject["Load CAD Geometry"].Arguments.set_state(
    {
        r"FileName": import_file_name,
        r"LengthUnit": r"mm",
        r"Refaceting": {
            r"Refacet": False,
        },
    }
)
meshing.workflow.TaskObject["Load CAD Geometry"].Execute()

The result was the following error message:
LookupError: Load CAD Geometry is not found at path /TaskObject

What is the problem and is there a solution?

Version information
ANSYS 2024R1
python 3.11

**Comment by mkundu1** [comment]
Can you please enable logging by calling `pyfluent.logging.enable()` immediately after importing pyfluent and share the file pyfluent.log generated in the working directory?
